### DISCUSSION TITLE
Number of particles per cell visible in the smokeview changes wih mesh sizes

### DISCUSSION THREAD

**Discussion by Ranhill** [firemodels/fds#12726] [Q&A] [answered]
Hi,

I am using particles to represent cone shaped vegetation with N_PARTICLES_PER_CELL=1. As N_PARTICLES_PER_CELL=1, I had assumed that even when I change the mesh size, the particles per cell will remain the same but that did not turn out to be true. Finer mesh had more visible particles than in the coarse mesh. Mass_per_volume remains the same. Should I change mass_per_volume as hit and trail to see more or less the uniform distribution of particles in the cone volume. Please guide! Thanks.

**Comment by mcgratta** [comment]
```
Finer mesh had more visible particles than in the coarse mesh. 
```
That is what I would expect.

**Reply by Ranhill** [reply]
Can you please tell the calculation behind it?

**Comment by mcgratta** [comment]
I do not understand what you are asking. In any event, `N_PARTICLES_PER_CELL` means "number of particles per FDS grid cell." If you were to cut the size of your (cubical) cells by a factor of 2, you will see 8 times as many particles. The mass that these particles represent would remain the same.

**Reply by Ranhill** [reply]
Thanks, I got it now!

**Comment by rmcdermo** [comment]
To clarify, the total mass of the particles in the cone volume remains the same under grid refinement (respecting the input MASS_PER_VOLUME).  The mass per particle decreases as you increase the number of particles (grid cells) in the volume.

**Reply by Ranhill** [reply]
Thanks for the clarification, I understand it now!

**Reply by Ranhill** [reply]
One more question: as the number of particles will decrease when meshes are made coarse, then that will also affect the drag??

**Comment by mcgratta** [comment, marked as answer]
All quantities associated with the particle are weighted. This includes the drag. There are verification cases discussed in the FDS User's and Verification Guides.
